aicoe/sesheta/utils.py

```diff
@@ -37,6 +37,7 @@
 _LOGGER.setLevel(logging.DEBUG if bool(os.getenv("DEBUG", False)) else logging.INFO)
 THOTH_DEVOPS_SPACE = os.getenv("SESHETA_THOTH_DEVOPS_SPACE", None)  # pragma: no cover
 AIOPS_DEVOPS_SPACE = os.getenv("SESHETA_AIOPS_DEVOPS_SPACE", None)  # pragma: no cover
+DISABLE_CHAT_NOTIFICATIONS = os.getenv("DISABLE_CHAT_NOTIFICATIONS", True)  # pragma: no cover
```

Comment:
Do we want the default to be `True`?

Asking because that makes it a breaking change